src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/DirectionTag.java

```diff
@@ -7,6 +7,7 @@
 
     BUILD_CASE("buildCase"),
     CASE_EDIT("caseEdit"),
+    CHANGE_DIRECTION_DUE_DATE("changeDirectionDueDate"),
```

Comment:
@tarun-palisetty We do not need new tag within this story because it is changing existing direction due dat not creating new direction